authors/apps/authentication/serializers.py

```diff
@@ -177,3 +177,38 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+class ForgetPasswordSerializer(serializers.Serializer):
+    """Serializer for forget password"""
+
+    email = serializers.CharField(max_length=255)
+
+
+class ResetPasswordSerializer(serializers.Serializer):
+    """Serializer for reset password"""
+
+    token = serializers.CharField(max_length=500)
```

Comment:
@mikenthiwa fetching the token from the URL is more convenient for the user and also reduces cases of human error. I would advise you to take this approach instead of having the user pass it as a JSON field.